br/pkg/restore/split_client.go

```diff
@@ -294,85 +294,99 @@ func splitRegionWithFailpoint(
 	})
 }
 
+func sendSplitRegionRequest(c *pdClient, ctx context.Context, regionInfo *RegionInfo, keys [][]byte, splitErrors *error, retry int) (bool, *kvrpcpb.SplitRegionResponse, error) {
```

Comment:
Nit: Moving `func sendSplitRegionRequest` after `func (c *pdClient) sendSplitRegionRequest` may make the diff more easy to review.